cmd/admin.go

```diff
@@ -538,6 +560,46 @@ func runDeleteUser(c *cli.Context) error {
 	return user_service.DeleteUser(user)
 }
 
+func runGenerateAccessToken(c *cli.Context) error {
+	if !c.IsSet("username") {
+		return fmt.Errorf("You must provide the username to generate a token for them")
+	}
+
+	ctx, cancel := installSignals()
+	defer cancel()
+
+	if err := initDB(ctx); err != nil {
+		return err
+	}
+
+	if err := storage.Init(); err != nil {
```

Comment:
I think we only need db access, so do we really need to init storage?

Reply:
Let me remove and try it

Reply:
Ok, it wasn't needed, I changed the PR , thanks for noticing it.